_posts/2024-05-23-wf-32-glow-blog-infoq.adoc

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: 'Blog: WildFly 32 Delivers Preview of Jakarta EE 11 and Final Version of WildFly Glow'
+date: 2024-05-23
+tags: wildfly glow galleon openshift provisioning
+synopsis: 
+author: jfdenise
+link: 
```

Comment:
could you please fill the `link` field in the front matter?

Reply:
Fixed